src/backends/github/implementation.js

```diff
@@ -94,7 +94,11 @@ export default class GitHub {
     return this.api.listFiles(this.config.get('media_folder'))
       .then(files => files.filter(file => file.type === 'file'))
       .then(files => files.map(({ sha, name, size, download_url, path }) => {
-        return { id: sha, name, size, url: download_url, path };
+        const url = new URL(download_url);
+        if (url.pathname.match(/.svg$/)) {
+          url.searchParams.append('sanitize', true);
```

Comment:
MS Edge does not support `searchParams`, and it seems to break the entire media library if a single SVG is added.

Reply:
It looks like Edge will support this in v17: https://caniuse.com/#feat=urlsearchparams. We normally support two versions, though, so it's not practical to just wait until it is released.

Reply:
These should be equivalent:
```js
url.searchParams.append('sanitize', true);
```
```js
url.search += (url.search.slice(1) === '' ? '?' : '&') + 'sanitize=true';
```

Reply:
@Jinksi @erquhart What do you think  about using my example code above instead? ^^^^^^^^^

Reply:
Good catch @tech4him1, agreed on the workaround.

Reply:
@tech4him1 @erquhart Updated https://github.com/netlify/netlify-cms/pull/954/commits/70c30905686c732f6b656063c4422a10f07bc2e8